app.json

```diff
@@ -18,6 +18,75 @@
     "EXTERNAL_URL": {
```

Comment:
I'm thinking

Either we can remove the optional variables from the list or we can add a default value for those env variables. Leaving empty might bring some confusion. what do you think?

Reply:
The intention is to allow the user to override these values, hence it is better to keep it. Default values could be set in the case of heroku.